eZ/Publish/SPI/Tests/FieldType/ImageIntegrationTest.php

```diff
@@ -124,6 +128,16 @@ public function getAliasCleanerMock(): MockObject
         return $this->aliasCleanerMock;
     }
 
+    private function getFilePathNormalizerMock(): IO\FilePathNormalizerInterface
+    {
+        if (!isset($this->filePathNormalizer)) {
```

Comment:
Must have missed it in the initial PR. This is a bad habit semantically, the class member is always "set". What you need to check is if it's `null ===`

Reply:
This will not be the case for PHP 7.4 with typed properties.

For
```php
private FilePathNormalizer $filePathNormalizer;
```
those two checks result in different results:
```php
$this->filePathNormalizer === null;
// An error occurs - we're accessing uninitialized property

isset($this->filePathNormalizer);
// Works normally
```

So no, in this case `isset` usage is preferred.

Reply:
> This will not be the case for PHP 7.4 with typed properties.
> 
> For
> 
> ```
> private FilePathNormalizer $filePathNormalizer;
> ```
> 
> those two checks result in different results:
> 
> ```
> $this->filePathNormalizer === null;
> // An error occurs - we're accessing uninitialized property
> 
> isset($this->filePathNormalizer);
> // Works normally
> ```
> 
> So no, in this case `isset` usage is preferred.

Interesting change. Thanks for pointing this out @Steveb-p.